packages/devextreme/js/__internal/ui/html_editor/html_editor.ts

```diff
@@ -412,94 +392,131 @@ class HtmlEditor extends Editor<Properties> {
 
   _renderScrollHandler(): void {
     const $scrollContainer = this._getContent();
-
     const initScrollData = prepareScrollData($scrollContainer);
-    // @ts-expect-error ts-error
-    eventsEngine.on($scrollContainer, addNamespace(scrollEvents.init, this.NAME), initScrollData, noop);
-    // @ts-expect-error ts-error
-    eventsEngine.on($scrollContainer, addNamespace(pointerEvents.move, this.NAME), this._pointerMoveHandler.bind(this));
+
+    eventsEngine.on(
+      $scrollContainer,
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      addNamespace(scrollEvents.init, this.NAME!),
+      initScrollData,
+      noop,
+    );
+
+    eventsEngine.on(
+      $scrollContainer,
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      addNamespace(pointerEvents.move, this.NAME!),
+      this._pointerMoveHandler.bind(this),
+    );
   }
 
   _applyTranscludedContent(): void {
-    const valueOption = this.option('value');
-    if (!isDefined(valueOption)) {
+    const { value } = this.option();
+
+    if (!isDefined(value)) {
       const html = this._deltaConverter.toHtml();
       const newDelta = this._quillInstance.clipboard.convert({ html });
 
       if (newDelta.ops.length) {
         this._quillInstance.setContents(newDelta);
+
         return;
       }
     }
 
     this._finalizeContentRendering();
   }
 
-  _hasTranscludedContent() {
-    // eslint-disable-next-line @typescript-eslint/prefer-optional-chain
-    return this._$templateResult && this._$templateResult.length;
+  _hasTranscludedContent(): boolean {
+    return Boolean(this._$templateResult?.length);
+  }
+
+  _getModulesConfig(): ModulesConfig {
+    const modulesConfig: ModulesConfig = extend(
+      {},
+      {
+        clipboard: this._getClipboardConfig(),
+        // TODO: extract some IE11 tweaks for the Quill uploader module
+        // dropImage: this._getBaseModuleConfig(),
+        imageCursor: this._getBaseModuleConfig(),
+        imageUpload: this._getModuleConfigByOption('imageUpload'),
+        keyboard: this._getKeyboardConfig(),
+        mentions: this._getModuleConfigByOption('mentions'),
+        multiline: Boolean(this.option('allowSoftLineBreak')),
+        resizing: this._getModuleConfigByOption('mediaResizing'),
+        table: true,
+        tableContextMenu: this._getModuleConfigByOption('tableContextMenu'),
+        tableResizing: this._getModuleConfigByOption('tableResizing'),
+        toolbar: this._getModuleConfigByOption('toolbar'),
+        uploader: this._getUploaderConfig(),
+        variables: this._getModuleConfigByOption('variables'),
+      },
+      this._getCustomModules(),
+    );
+
+    return modulesConfig;
   }
 
-  _getModulesConfig() {
+  _getUploaderConfig(): {
```

Comment:
[naming]
```suggestion
  _getUploaderModuleConfig(): {
```
I suggest you to update it for all similar modules to point explicitly that it's a config for module. "uploaderConfig" sound like a config for subcomponent, "keyboardConfig" sounds just strange)

Reply:
[081e5880faafd04de7b57955e52c34b00540bbbb]